internal/blocktx/store/postgresql/stats.go

```diff
@@ -0,0 +1,30 @@
+package postgresql
+
+import (
+	"context"
+
+	"github.com/bitcoin-sv/arc/internal/blocktx/store"
+)
+
+func (p *PostgreSQL) GetStats(ctx context.Context) (*store.Stats, error) {
+	q := `
+	SELECT count(*) FROM (
+	SELECT unnest(ARRAY(
+			SELECT a.n	
+			FROM generate_series((SELECT max(height) - 2016 AS block_height FROM blocktx.blocks b), (SELECT max(height) AS block_height FROM blocktx.blocks b)) AS a(n)
```

Comment:
Why 2016? 

It would be better to have this as a const in the top of the file rather than directly in the query